nilearn/reporting/get_clusters_table.py

```diff
@@ -371,9 +375,14 @@ def get_clusters_table(
         # Now re-label and create table
         label_map = label(binarized, bin_struct)[0]
         clust_ids = sorted(np.unique(label_map)[1:])
-        peak_vals = np.array(
-            [np.max(temp_stat_map * (label_map == c)) for c in clust_ids]
-        )
+        if not two_sided and stat_threshold < 0:
+            peak_vals = np.array(
+                [np.min(temp_stat_map * (label_map == c)) for c in clust_ids]
```

Comment:
Don't we ned to re-flip the sign ?

Reply:
given that we are working on `temp_stat_map` and not on `stat_map` we are fine, no?

Reply:
let me double check

Reply:
added a check in the test to make sure the content of the original image is unchanged